### ISSUE TITLE
Todo: Get a 530 Wiki Page Dump

### ISSUE THREAD

**Issue by dginovker** [2009scape/Dumps#1]
This gent has provided the code and a howto: https://www.rune-server.ee/runescape-development/rs-503-client-server/downloads/695932-offline-wiki-dumps-550-634-667-718-742-a.html

**Comment by ryannathans** [comment]
i've actually been writing my own scraper and pulling from the live wiki from 2009, i'm scraping the wikimedia markup - not the html

**Comment by dginovker** [comment]
Whoops, missclicked close. That's pretty sick, want to dump when you're done?

**Comment by ryannathans** [comment]
Sure, need someone to host a wikimedia so it's publicly editable. I don't really know how wikiwedia works in detail, maybe I need to scrape the templates too?

**Comment by dginovker** [comment]
Yeah I mean someone can throw up a wikimedia anytime. I just figured step one is get the dump while it's still accessible :P
